src/Parser/RestoDruid/Modules/Traits/Grovewalker.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+import Module from 'Parser/Core/Module';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import calculateEffectiveHealing from 'Parser/Core/calculateEffectiveHealing';
+import SPELLS from 'common/SPELLS';
+
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import { formatPercentage } from 'common/format';
+import {HEALS_MASTERY_STACK} from '../../Constants';
+
+
+const GROVEWALKER_HEALING_INCREASE = 0.01;
+
+/**
+ * Grovewalker (Artifact Trait)
+ * Increases all healing over time you do by 1%
+ */
+class Grovewalker extends Module {
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  rank = 0;
+  healing = 0;
+
+  on_initialized() {
+    this.rank = this.combatants.selected.traitsBySpellId[SPELLS.GROVEWALKER.id];
+    this.active = this.rank > 0;
+  }
+
+  on_byPlayer_heal(event) {
+    const spellId = event.ability.guid;
+    if(!HEALS_MASTERY_STACK.indexOf(spellId)) {
```

Comment:
This isn't the correct check, `HEALS_MASTERY_STACK.indexOf(spellId)` evaluates false for the item in index 0. Use HEALS_MASTERY_STACK.includes(spellId) instead.

Reply:
Done